apps/andi/lib/andi_web/live/dataset_live_view.ex

```diff
@@ -25,27 +25,53 @@ defmodule AndiWeb.DatasetLiveView do
           </div>
         </form>
       </div>
-      <%= live_component(@socket, Table, datasets: @datasets) %>
+      <%= live_component(@socket, Table, id: :datasets_table, datasets: @datasets, order: @order) %>
     </div>
     """
   end
 
   def mount(_session, socket) do
-    {:ok, assign(socket, datasets: nil, search_text: "")}
+    {:ok, assign(socket, datasets: nil, search_text: nil, order: {"data-title", "asc"}, params: %{})}
+  end
+
+  def handle_info({:order, field}, socket) do
+    order_dir =
+      case socket.assigns.order do
+        %{^field => "asc"} -> "desc"
```

Comment:
Still curious about this guy.

Reply:
Nevermind. I didn't refresh and see Jeff's explination.

Reply:
I can't see Jeff's explanation